containers/unit_tests/TestScatterView.hpp

```diff
@@ -259,11 +259,11 @@ struct test_scatter_view_impl_cls<DeviceType, Layout, Duplication, Contribution,
       auto val0 = host_view(i, 0);
       auto val1 = host_view(i, 1);
       auto val2 = host_view(i, 2);
-      EXPECT_TRUE(std::fabs((val0 - 65536.0) / 65536.0) < 1e-14)
+      EXPECT_LT(std::fabs((val0 - 65536.0) / 65536.0), 1e-14)
```

Comment:
I think `EXPECT_NEAR` would be more appropriate in these cases.  [GoogleTest Assertions](https://google.github.io/googletest/reference/assertions.html)

Reply:
6684cda8b4f826e7913b77a475a1877cc3a8aa9e